ingress/elb/elb.go

```diff
@@ -65,18 +75,33 @@ func (e *elb) Attach(frontend types.FrontendInput) (int, error) {
 		return 0, err
 	}
 
+	// Save these now so we can always know what we might have done
+	// up until this point we have only read data
+	e.elbs = clusterFrontEnds
+	e.instanceID = instance
+	registered := 0
+
+	if len(clusterFrontEnds) != e.expectedFrontends {
+		log.Warnf("Expected to find %d elb frontends, only found %d", e.expectedFrontends, len(clusterFrontEnds))
+	}
```

Comment:
If this isn't going to fail at startup, we should report it in a Health() status. That way kubernetes will try to restart / reattach it.